src/formats/git.cpp

```diff
@@ -127,7 +128,7 @@ std::string GitCommitLog::logCommand() {
     return log_command;
 }
 
-GitCommitLog::GitCommitLog(const std::string& logfile) : RCommitLog(logfile, 'u') {
+GitCommitLog::GitCommitLog(const std::string& logfile) : RCommitLog(logfile, 'u'), m_repository_path(logfile) {
```

Comment:
Maybe instead RCommitLog could keep a copy of logfile and then we can refer to it later with self.logfile